display_list/display_list_builder.h

```diff
@@ -271,11 +281,13 @@ class DisplayListBuilder final : public virtual Dispatcher,
   }
 
   struct LayerInfo {
-    LayerInfo(bool has_layer = false)
-        : has_layer(has_layer),
+    LayerInfo(size_t save_layer_offset = 0, bool has_layer = false)
+        : save_layer_offset(save_layer_offset),
+          has_layer(has_layer),
           cannot_inherit_opacity(false),
           has_compatible_op(false) {}
 
+    size_t save_layer_offset;
```

Comment:
can you add a comment here about what this is? it's not obvious from reading the code.

Reply:
Big comment added...